src/EnergyPlus/PlantHeatExchangerFluidToFluid.cc

```diff
@@ -827,20 +831,47 @@ namespace PlantHeatExchangerFluidToFluid {
 		// second deal with Loop Demand Side
 		tmpDmdSideDesignVolFlowRate = FluidHX( CompNum ).DemandSideLoop.DesignVolumeFlowRate;
 		if ( FluidHX( CompNum ).DemandSideLoop.DesignVolumeFlowRateWasAutoSized ) {
-			if ( tmpSupSideDesignVolFlowRate > SmallWaterVolFlow ) {
-				tmpDmdSideDesignVolFlowRate = tmpSupSideDesignVolFlowRate;
-				if ( PlantFirstSizesOkayToFinalize ) FluidHX( CompNum ).DemandSideLoop.DesignVolumeFlowRate = tmpDmdSideDesignVolFlowRate;
-			} else {
-				tmpDmdSideDesignVolFlowRate = 0.0;
-				if ( PlantFirstSizesOkayToFinalize ) FluidHX( CompNum ).DemandSideLoop.DesignVolumeFlowRate = tmpDmdSideDesignVolFlowRate;
-			}
-			if ( PlantFinalSizesOkayToReport ) {
-				ReportSizingOutput( "HeatExchanger:FluidToFluid", FluidHX( CompNum ).Name,
-					"Loop Demand Side Design Fluid Flow Rate [m3/s]", FluidHX( CompNum ).DemandSideLoop.DesignVolumeFlowRate );
+			if (PltSizNumDmdSide > 0) {
+				if ( tmpSupSideDesignVolFlowRate > SmallWaterVolFlow ) {
+					// calculate the supply side capacity per volume flow rate
+					tmpDeltaTSupLoop = PlantSizData(PltSizNumSupSide).DeltaT;
+					Cp = GetSpecificHeatGlycol(PlantLoop(FluidHX(CompNum).SupplySideLoop.LoopNum).FluidName, DataGlobals::InitConvTemp, PlantLoop(FluidHX(CompNum).SupplySideLoop.LoopNum).FluidIndex, RoutineName);
+					rho = GetDensityGlycol(PlantLoop(FluidHX(CompNum).SupplySideLoop.LoopNum).FluidName, DataGlobals::InitConvTemp, PlantLoop(FluidHX(CompNum).SupplySideLoop.LoopNum).FluidIndex, RoutineName);
```

Comment:
For HW loops this should use HWInitConvTemp. Is this HX allowed in cold or hot water loops? If so then the loop type would need to be tested to use the correct InitConvTemp initialization.